File: extensions/services/pg2luciddb/samples/csharp/connect.cs
using System;
using System.Data;
using Npgsql;

class PostgreSQL2LucidDBTest
{
    static public void Main()
    {
        // TODO: change to your real values
        string connString = "Server=127.0.0.1; Port=9999; Database=LOCALDB; User Id=sa; Password=";

        NpgsqlConnection conn = null;
        try
        {
            conn = new NpgsqlConnection(connString);
            conn.Open();
            Console.WriteLine("Established connection to PG2LucidDB");
            // get server version:
            NpgsqlCommand comm = conn.CreateCommand();
            comm.CommandText = "SELECT VERSION()";
            Console.WriteLine("Server version: {0}", comm.ExecuteScalar());
        }
        catch(Exception ex)
        {
            Console.WriteLine("Exception occured: {0}", ex);
        }
        finally
        {
            if (conn != null)
            {
                conn.Close();
            }
        }       
    }
}

File: extensions/services/pg2luciddb/samples/php/connect.php
<?php

 // TODO: change your connection string here
 $connectionString = "host=localhost port=9999 dbname=LOCALDB user=sa password=";

 // open connection to the server
 $connection = pg_connect($connectionString) or die ("Unable to connect to PG2LucidDB: " . pg_last_error($connection)); 

 echo("Established connection to PG2LucidDB\n");

 // get server version:
 $result = pg_query("select version()");

 if ($result)
 {
     $row = pg_fetch_row($result);
     echo("Server version: $row[0]\n");
     pg_free_result($result);
 }

 // close connection:
 pg_close($connection);  

?>